apps/mobile/src/app/(home)/settings/notifications/_layout.tsx

```diff
@@ -0,0 +1,66 @@
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+import { BackButtonHeader } from '@/components/headers/back-button';
+import { SimpleHeader } from '@/components/headers/containers/simple-header';
+import { TitleHeader } from '@/components/headers/title';
+import { TabBar } from '@/components/tab-bar';
+import { AppRoutes } from '@/routes';
+import { t } from '@lingui/macro';
+import { Tabs, router, usePathname, useRouter } from 'expo-router';
+
+function HeaderBottomTabs() {
```

Comment:
Maybe `HeaderBottomTabs` + `NavigationHeader` could be in their own `Header` component?

Reply:
Our headers really need some work, so I'll see if there is an issue to coordinate all these a bit better. If not, I'll make one.

Reply:
Thanks @fbwoolf 👍 It seems like we have the headers we need but they could do with some refactoring